research/dao/dao-research.md

```
@@ -0,0 +1,133 @@
+
+The U.S. Securities and Exchange Commission (SEC) has declared that selling crypto tokens in a DAO requires registering them as securities, making them subject to federal security laws. The SEC requests that issuers of distributed ledger or blockchain tech-based securities must register offers and sales unless a valid exception applies. Those participating in unregistered offerings may also be liable for violations of securities laws.
+
+The purpose of these registration provisions is to ensure that investors are sold investments that include all the proper disclosures and are subject to regulatory scrutiny for investor protection.
+
+---
+
+# 2. Token Distribution Model for Farmers and Stakeholders
+
+This section is crucial for a crypto project’s tokenomics as it determines the token allocation to different groups of investors. Savvy investors analyze the fairness of the distribution structure. An ideal allocation has a small portion allocated to insiders, with the rest split among other groups.
+
+### Breakdown of Typical Token Distributions
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)